pkg/server/commands/list_objects.go

```diff
@@ -219,16 +220,17 @@ func (q *ListObjectsQuery) Execute(
 	}
 }
 
+// ExecuteStreamed executes the ListObjectsQuery, returning a stream of object IDs.
+// It ignores the value of q.ListObjectsMaxResults and returns all available results
+// until q.ListObjectsDeadline is hit
 func (q *ListObjectsQuery) ExecuteStreamed(
 	ctx context.Context,
 	req *openfgapb.StreamedListObjectsRequest,
 	srv openfgapb.OpenFGAService_StreamedListObjectsServer,
 ) error {
 
-	resultsChan := make(chan string, 1)
-	if q.ListObjectsMaxResults > 0 {
-		resultsChan = make(chan string, q.ListObjectsMaxResults)
-	}
+	maxResults := uint32(math.MaxUint32)
```

Comment:
I'm not doing `q.ListObjectsMaxResults = math.maxuint32` because i don't want this function to have the side effect of changing `q`. As discussed in prior PRs, in the future we would like to have the ability to create one `query`/`command` object globally and use that across requests.